<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TranslationWidget</class>
 <widget class="QWidget" name="TranslationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>翻译文件管理器</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>文件选择</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="btnNewSource">
        <property name="text">
         <string>新建源文件</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnSelectSource">
        <property name="text">
         <string>选择源文件</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnNewTarget">
        <property name="text">
         <string>新建目标文件</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btnSelectTarget">
        <property name="text">
         <string>选择目标文件</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QLineEdit" name="lineEditSource">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>未选择源文件</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>目标文件列表</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListWidget" name="listWidgetTargets"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="btnRemoveTarget">
          <property name="text">
           <string>移除选中</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnClearTargets">
          <property name="text">
           <string>清空列表</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>翻译内容</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="editTriggers">
         <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectionBehavior::SelectItems</enum>
        </property>
        <column>
         <property name="text">
          <string>Key</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>源翻译</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>操作</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btnAddKey">
        <property name="text">
         <string>添加Key</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDeleteKey">
        <property name="text">
         <string>删除选中Key</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSaveSource">
        <property name="text">
         <string>保存源文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSaveTarget">
        <property name="text">
         <string>保存目标文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSaveAll">
        <property name="text">
         <string>一键保存所有</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
